frontend/src/assets/css/padding-variables.css

```diff
@@ -0,0 +1,7 @@
+:root {
+    --p-025: 0.25rem;
```

Comment:
The same as with margins

Reply:
Done in commit [b72b2bf](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/24/commits/b72b2bf44b80194ef93f40a62fa9cc92f9bb6024)